frontends/main/src/app-pages/AboutPage/AboutPage.test.tsx

```diff
@@ -1,20 +1,10 @@
-import { renderTestApp, screen, waitFor, setMockResponse } from "@/test-utils"
-import { urls } from "api/test-utils"
-import * as commonUrls from "@/common/urls"
-import { Permissions } from "@/common/permissions"
+import React from "react"
+import { screen, renderWithProviders } from "@/test-utils"
+import { AboutPage } from "./AboutPage"
 
 describe("AboutPage", () => {
   test("Renders title", async () => {
-    setMockResponse.get(urls.userMe.get(), {
-      [Permissions.Authenticated]: true,
-    })
-
-    renderTestApp({
-      url: commonUrls.ABOUT,
-    })
-    await waitFor(() => {
-      expect(document.title).toBe("About Us | MIT Learn")
-    })
```

Comment:
I removed this and some other metadata tests since the metadata is now handled by NextJS.

It probably is worth having metadata tests, but they should be added in https://github.com/mitodl/hq/issues/5410